trim_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class TrimDecorator < Base_decorator
```

Comment:
- [ ] The same typo is present here:

```suggestion
class TrimDecorator < BaseDecorator
```